pkg/types/network/change/types.proto

```diff
@@ -0,0 +1,82 @@
+/*
+Copyright 2019-present Open Networking Foundation.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+syntax = "proto3";
+
+package onos.config.network.change;
+
+import "google/protobuf/timestamp.proto";
+import "gogoproto/gogo.proto";
+import "github.com/onosproject/onos-config/pkg/types/device/change/types.proto";
+
+// Status is the status of a network change
+enum Status {
```

Comment:
These appear in all your proto files - could they be reused from one file?

Reply:
I don't have a problem with using a single pair of enums. The reason they're implemented this way is because creating a separate status/reason enum for each type allows them to evolve separately if needed. If one pair of enums applies to all types then any change to the enums also needs to be valid for all types.